test/controllers/api/v2/dlmlocks_controller_test.rb

```diff
@@ -124,6 +124,30 @@ class Api::V2::DlmlocksControllerTest < ActionController::TestCase
         assert_equal host2, dlmlock.reload.host
       end
     end
+
+    context '#events' do
+      test 'should return the dlmlock_events for a given dlm_lock' do
+        expected_size = 3
+
+        # A random dlmlock with events, that should not be in the response.
+        dlmlock1 = FactoryBot.create(:dlmlock, :host => host1)
+        FactoryBot.create_list(:dlmlock_event, 2, dlmlock: dlmlock1)
+
+        dlmlock = FactoryBot.create(:dlmlock, :host => host1)
+        FactoryBot.create_list(:dlmlock_event, expected_size, dlmlock: dlmlock)
+
+        expected_ids = dlmlock.dlmlock_events.pluck(:id).sort
+        expected_keys = ['id', 'event_type', 'created_at', 'updated_at']
+
+        get :events, params: { id: dlmlock.id }
+        assert_response :success
+
+        body = ActiveSupport::JSON.decode(@response.body)
+        assert_equal expected_size, body.count
```

Comment:
This should be `assert_equal expected_size, body['total']` to be consistend with other api resonses.

```json
{
  "total": 3,
  "subtotal": 3,
  "page": 1,
  "per_page": 20,
  "search": "",
  "sort": {
    "by": null,
    "order": null
  },
  "results": [{"created_at":"2017-08-24 12:11:31 +0200","updated_at":"2018-07-13 14:34:34 +0200","id":1,"host_id":74,"name":"test","enabled":true,"type":"ForemanDlm::Dlmlock::Update","host":{"name":"kim-regnier.example.com"}},{"created_at":"2017-08-24 14:03:34 +0200","updated_at":"2018-06-26 11:22:49 +0200","id":2,"host_id":null,"name":"test fsfds","enabled":false,"type":"ForemanDlm::Dlmlock::Update","host":null},{"created_at":"2018-07-13 14:56:34 +0200","updated_at":"2018-07-13 14:56:34 +0200","id":3,"host_id":74,"name":"update_spaceship","enabled":true,"type":null,"host":{"name":"kim-regnier.example.com"}}]
}
```